lib/docs/scrapers/rxjs.rb

```diff
@@ -0,0 +1,94 @@
+require 'yajl/json_gem'
+
+module Docs
+  class Rxjs < UrlScraper
+    self.name = 'RxJS'
+    self.type = 'rxjs'
+    self.links = {
+      home: 'https://rxjs-dev.firebaseapp.com/',
+      code: 'https://github.com/ReactiveX/rxjs'
+    }
+
+    options[:max_image_size] = 256_000
+
+    options[:attribution] = <<-HTML
+      &copy; 2015&ndash;The RxJS Project Developers<br>
+      Code licensed under an Apache-2.0 License. Documentation licensed under CC BY 4.0.
```

Comment:
@benlesh can you confirm if this attribution is correct?

I took the year from the first RxJS release (https://github.com/ReactiveX/rxjs/releases/tag/0.0.0-prealpha.1) and the license text from https://rxjs-dev.firebaseapp.com/.